src/grid/storage.rs

```diff
@@ -20,6 +20,8 @@ pub struct Storage<T> {
     inner: Vec<T>,
     zero: usize,
     visible_lines: Line,
+    #[serde(skip)]
+    len: usize,
```

Comment:
Please add a comment about the semantics of `len` in this case. I have an assumption from working with `std::collections` that `len` will be the total number of reachable/usable/active elements in this collection.

Reply:
Tried documenting the use of it a bit better, hopefully the wording makes sense like this.